### PR TITLE
[Keyboard] Support for dumbpad_v3x

### PR DESCRIPTION
**Pull request by dickofn**
## Description

This change is the continuity of this PR https://github.com/qmk/qmk_firmware/pull/16285, all recommendations in the conversation have already been applied.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* https://github.com/qmk/qmk_firmware/pull/16285

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dickofn** [comment]
Just want to state, full credit goes to @deveth0 and @imchipwood 

@noroadsleft this is the completed version of the PR that we discussed earlier https://github.com/qmk/qmk_firmware/pull/16285#issuecomment-1272847731

File: keyboards/dumbpad/v3x/config.h [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER      0x0010
+
```
Comment (noroadsleft) [inline keyboards/dumbpad/v3x/config.h:24]: We're in the process of moving USB Device Descriptors to `info.json`.

```suggestion
```

File: keyboards/dumbpad/v3x/info.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "dumbpad/v3x",
+    "url": "https://www.github.com/imchipwood/dumbpad",
+    "maintainer": "imchipwood",
+    "layouts": {
```
Comment (noroadsleft) [inline keyboards/dumbpad/v3x/info.json:5]:
```suggestion
    "usb": {
        "device_version": "0.1.0"
    },
    "layouts": {
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by dickofn** [comment]
pushed new commit for requested change

**Comment by dickofn** [comment]
@noroadsleft may I change it to draft half way? I want to try vial support

File: keyboards/dumbpad/v3x/config.h (hunk 1 above)
Comment (dickofn) [reply keyboards/dumbpad/v3x/config.h]: commit: https://github.com/qmk/qmk_firmware/pull/18663/commits/2086c250fab2950ce747b724ffd78b7e23d44a36

File: keyboards/dumbpad/v3x/info.json (hunk 2 above)
Comment (dickofn) [reply keyboards/dumbpad/v3x/info.json:8]: commit: https://github.com/qmk/qmk_firmware/pull/18663/commits/a1f515226355d71875a8961db22f8a458d7dde23

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**